lib/matplotlib/tri/tricontour.py

```diff
@@ -256,23 +249,30 @@ def _contour_args(self, args, kwargs):
 antialiased : bool, optional
     Enable antialiasing, overriding the defaults.  For
     filled contours, the default is *True*.  For line contours,
-    it is taken from :rc:`lines.antialiased`.
-
-nchunk : int >= 0, optional
-    If 0, no subdivision of the domain.  Specify a positive integer to
-    divide the domain into subdomains of *nchunk* by *nchunk* quads.
-    Chunking reduces the maximum length of polygons generated by the
-    contouring algorithm which reduces the rendering workload passed
-    on to the backend and also requires slightly less RAM.  It can
-    however introduce rendering artifacts at chunk boundaries depending
-    on the backend, the *antialiased* flag and value of *alpha*.""")
+    it is taken from :rc:`lines.antialiased`.""")
 
 
 @docstring.Substitution(func='tricontour', type='lines')
 @docstring.dedent_interpd
 def tricontour(ax, *args, **kwargs):
     """
     %(_tricontour_doc)s
+    
```

Comment:
There is a linting failure here due to some unnecessary whitespace at the start of the line.

Reply:
I removed the whitespaces. Please take a look and tell me if there's something to be added